build/Targets/Versions.props

```diff
@@ -12,9 +12,9 @@
 
   <PropertyGroup>
     <!-- This is the assembly version of Roslyn from the .NET assembly perspective. It should only be revved during significant point releases. -->
-    <RoslynAssemblyVersionBase Condition="'$(RoslynAssemblyVersion)' == ''">2.8.0</RoslynAssemblyVersionBase>
+    <RoslynAssemblyVersionBase Condition="'$(RoslynAssemblyVersion)' == ''">2.8.1</RoslynAssemblyVersionBase>
```

Comment:
We don't bump assembly version on minor releases. File version only.

Reply:
OK, will fix that.